src/display/api.js

```diff
@@ -2582,14 +2626,14 @@ class RenderTask {
      * Callback for incremental rendering -- a function that will be called
      * each time the rendering is paused.  To continue rendering call the
      * function that is the first argument to the callback.
-     * @type {function}
+     * @type {() => {}}
```

Comment:
Without knowing anything about TypeScript, this does look a bit weird; is this actually correct!?

Reply:
Hmm - it _is_ a valid typescript type (as far as I remember ;) And I put it there because it was complaining. Now I put back `function`, and it's OK. Go figure...